.pre-commit-config.yaml

```diff
@@ -41,6 +41,7 @@ repos:
     rev: v0.32.0
     hooks:
       - id: yapf
+        args: ['-i', '--style={based_on_style: google, spaces_around_power_operator: true, column_limit: 100}']
```

Comment:
Column limit should be around 80 characters, 100 might be a bit high. Also, not sure what other change was made, but it made `amp_cli.py` a bit less readable IMO.

Reply:
Removed.